sql/base.sql

```diff
@@ -1036,99 +1040,104 @@ INSERT OR IGNORE INTO RequirementArguments
-	VALUES ('LIMES_SIEGE_INDUSTRIAL_ERA_CPLMOD' , 'UnitPromotionClass' , 'PROMOTION_CLASS_SIEGE' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_INDUSTRIAL_ERA_CPLMOD' , 'EraType' , 'ERA_INDUSTRIAL' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_INDUSTRIAL_ERA_CPLMOD' , 'Amount' , '100' , '-1');
-	
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_MODERN_ERA_CPLMOD' , 'UnitPromotionClass' , 'PROMOTION_CLASS_SIEGE' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_MODERN_ERA_CPLMOD' , 'EraType' , 'ERA_MODERN' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_MODERN_ERA_CPLMOD' , 'Amount' , '100' , '-1');
-	
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_ATOMIC_ERA_CPLMOD' , 'UnitPromotionClass' , 'PROMOTION_CLASS_SIEGE' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_ATOMIC_ERA_CPLMOD' , 'EraType' , 'ERA_ATOMIC' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_ATOMIC_ERA_CPLMOD' , 'Amount' , '100' , '-1');
-	
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_INFORMATION_ERA_CPLMOD' , 'UnitPromotionClass' , 'PROMOTION_CLASS_SIEGE' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_INFORMATION_ERA_CPLMOD' , 'EraType' , 'ERA_INFORMATION' , '-1');
-INSERT OR IGNORE INTO ModifierArguments (ModifierId , Name , Value , Extra)
-	VALUES ('LIMES_SIEGE_INFORMATION_ERA_CPLMOD' , 'Amount' , '100' , '-1');
-
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_ANCIENT_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_CLASSICAL_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_MEDIEVAL_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_RENAISSANCE_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_INDUSTRIAL_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_MODERN_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_ATOMIC_ERA_CPLMOD');
-INSERT OR IGNORE INTO PolicyModifiers (PolicyType , ModifierId)
-	VALUES ('POLICY_LIMES' , 'LIMES_SIEGE_INFORMATION_ERA_CPLMOD');
+INSERT OR IGNORE INTO Types
```

Comment:
I think there should be some comments/documentation regarding that this is where the seige cards are created

Reply:
Documentation is imperative for organization and future edits

Reply:
yes, just add a comment to the top that says `--add 3 siege policy cards`

also, please delete the 2 lines that have to do with limes being obsolete. It was only needed when siege production was attached to it.